trillian/ctfe/instance.go

```diff
@@ -177,3 +206,25 @@ func ValidateLogMultiConfig(cfg *configpb.LogMultiConfig) (map[string]*configpb.
 
 	return backendMap, nil
 }
+
+// ToMultiLogConfig creates a multi backend config proto from the data
+// loaded from a single backend configuration file. All the log configs
```

Comment:
maybe "single-backend configuration file" so it's harder to read this as "single (backend configuration file)"

Reply:
Done.

Reply:
Is it?

Reply:
I changed the one in main.go and missed this. Now it's done.